docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md

```diff
@@ -119,9 +119,9 @@ For more information, read the
    azure_storage_account_key="Please set me up!"
    ```
 
-1. Finally, enter credentials for your chosen destination as per the [docs](../destinations/).
+2. Finally, enter credentials for your chosen destination as per the [docs](../destinations/).
 
-1. You can pass the bucket URL and glob pattern or use `config.toml`. For local filesystems, use
+3. You can pass the bucket URL and glob pattern or use `config.toml`. For local filesystems, use
    `file://` or skip the schema and provide the local path in a format native for your operating system.
```

Comment:
Just make this a bit more clear. I understand that for local filesystem, I have to use `file://` or the code shown below would also work. Just make it a bit more clear.

Reply:
I would say you can use `file://` as follows:
- code example

or 
Define it as follows:
```
[sources.filesystem] # use [sources.readers.credentials] for the "readers" source
bucket_url='~\Documents\csv_files\'
file_glob="*"
```

Reply:
updated as per comments